core/crypto/commitments/commitments_test.go

```diff
@@ -45,3 +49,39 @@ func TestCommit(t *testing.T) {
 		}
 	}
 }
+
+func TestVectors(t *testing.T) {
+	// Use a constant key of zero to obtain consistent test vectors.
+	// Real commitment library MUST use random keys.
+	Rand = dev.Zeros
+	zeroKey := []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
+	for _, tc := range []struct {
+		userID, appID, data string
+		want                []byte
+	}{
+		{"foo", "app", "bar", dh("064c8933f50f897e8b179065c6b3ec13e9d093337c6d403c77e3ed1701378ed6")},
```

Comment:
We should expect the same `want` for all input, right? Can we add a couple other cases with different input and same `want`?